tools/start_server.py

```diff
@@ -377,40 +405,24 @@ def uninstall_app(self, _app_id):
       }
     }
 
-  @staticmethod
-  def start_file_server(_host, _port, _remote_host=None, _remote_port=None):
-    WebEngineManager.file_server = WebengineFileServer(_host, _port, _remote_host, _remote_port)
-    thd = threading.Thread(target=WebEngineManager.file_server.start)
-    thd.start()
-
-  @staticmethod
-  def stop_file_server():
-    if WebEngineManager.file_server is not None:
-      WebEngineManager.file_server.stop()
-      WebEngineManager.file_server = None
-
 def main():
+  parser =  argparse.ArgumentParser(description="Handle backend operations for the hmi")
+  parser.add_argument('--host', type=str, required=True, help="Backend server hostname")
+  parser.add_argument('--port', type=int, required=True, help="Backend server port number")
+  parser.add_argument('--fs-port', type=int, default=4000, help="File server port number")
+  parser.add_argument('--remote-fs-host', type=str, help="File server's remote hostname(to be sent back to the client hmi)")
+  parser.add_argument('--remote-fs-port', type=int, help="File server's remote port number(to be sent back to the client hmi)")
 
-  if len(sys.argv) < 3:
-    print('\033[31;01mMissing required arguments: hostname and port\033[0m')
-    sys.exit(1)
+  args = parser.parse_args()
+  Flags.FILE_SERVER_HOST = args.host
+  Flags.FILE_SERVER_PORT = args.fs_port
+  Flags.FILE_SERVER_REMOTE_HOST = args.remote_fs_host if args.remote_fs_host else Flags.FILE_SERVER_HOST
+  Flags.FILE_SERVER_REMOTE_PORT = args.remote_fs_port if args.remote_fs_port else Flags.FILE_SERVER_PORT
```

Comment:
I think FILE_SERVER_REMOTE_HOST and FILE_SERVER_REMOTE_PORT can be combined since they are only used together to tell clients the URL where the server is. could be FILE_SERVER_URI...

Reply:
Good idea. Added in https://github.com/smartdevicelink/generic_hmi/pull/273/commits/44fc9bf4393cefc756724463107b6b846c44a115